nodeup/pkg/model/context.go

```diff
@@ -693,3 +693,7 @@ func (c *NodeupModelContext) CNIBinDir() string {
 func (c *NodeupModelContext) CNIConfDir() string {
 	return "/etc/cni/net.d/"
 }
+
+func (c *NodeupModelContext) InstallNvidiaRuntime() bool {
```

Comment:
Maybe `UseNvidiaGPU` would be a better name:
```suggestion
func (c *NodeupModelContext) UseNvidiaGPU() bool {
```

Reply:
What we do is:
* install it
* configure it
* set it to be used by default by containerd

But from kOps perspective, we don't really use it. Containerd will most likely use it, but technically a user can choose not to.

Reply:
It doesn't install it either, it sets it up.
You can see it also as the desire to use the local CPU with k8s.

Reply:
I am not sure I follow this one. This function determines if the driver and runtime packages are installed. As in getting `apt-get install`ed. It is used to configure containerd but that is a smallish part of it.

Reply:
This function should check if Nvidia "stuff" is enabled or not, not if installed. Install is just a consequence of it being enabled and supported by the hardware.